models/sales/intermediate/int_hubspot__deals_enhanced.sql

```diff
@@ -44,11 +59,17 @@ with deals as (
         on deals.deal_pipeline_id = pipelines.deal_pipeline_id
     left join pipeline_stages 
         on deals.deal_pipeline_stage_id = pipeline_stages.deal_pipeline_stage_id
+    left join aggregate_merged_deals
+        on deals.deal_id = aggregate_merged_deals.deal_id
+    left join merged_deals
+        on deals.deal_id = merged_deals.merged_deal_id
```

Comment:
Would you be able to elaborate on the need for this join?

Reply:
This was to be used to check which deals were merged. But with the update to the merged deal exclusion filter, I don't think I'll need this, so removing